### ISSUE TITLE
Accordion Menu Prop Bug

### ISSUE THREAD

**Issue by andytudhope** [makerdao/community-portal#58]
Content creators should be able to use the Accordions with their own unique open/close icons. Currently this fails:

```
<Accordion openIcon="heart" closeIcon="brokenHeart">

Of course, it's also possible to have your own icons for opening and closing

</Accordion>
```

Does not render any icon when closed, and defaults to the `-` icon when open, despite whatever you pass it.

**Comment by andytudhope** [comment]
Update here: the issue was because the icon `heart` and `brokenHeart` are not in the icon pack. If I change the code to:

```
<Accordion openIcon="chevron_down" closeIcon="chevron_up">

Of course, it's also possible to have your own icons for opening and closing

</Accordion>
```

The openIcon works as expected, but the closeIcon still defaults to the `-` rather than the specified chevron here.

**Comment by MaximumCrash** [comment]
Fixed by #66
